playground/worker/__tests__/iife/worker-iife.spec.ts

```diff
@@ -116,31 +120,29 @@ test('module worker', async () => {
   await untilUpdated(
     async () => page.textContent('.worker-import-meta-url'),
     /A\sstring.*\/iife\/.+url-worker\.js.+url-worker\.js/,
-    true,
   )
   await untilUpdated(
     () => page.textContent('.worker-import-meta-url-resolve'),
     /A\sstring.*\/iife\/.+url-worker\.js.+url-worker\.js/,
-    true,
   )
   await untilUpdated(
     () => page.textContent('.worker-import-meta-url-without-extension'),
     /A\sstring.*\/iife\/.+url-worker\.js.+url-worker\.js/,
-    true,
   )
   await untilUpdated(
     () => page.textContent('.shared-worker-import-meta-url'),
     'A string',
-    true,
   )
 })
 
 test('classic worker', async () => {
   await untilUpdated(() => page.textContent('.classic-worker'), 'A classic')
```

Comment:
Some assertions here work on test-build, but not running.

Reply:
Linking reference: https://github.com/vitejs/vite/blob/3d86edb7ac3da7d396507dbe97028b058f59547b/playground/worker/classic-worker.js#L18-L20